### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by dakkusingh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,8 +25,7 @@ COPY install-composer.sh /usr/local/bin/
 RUN install-composer.sh
 
 # Install Robo CI.
-# @TODO replace the following URL by http://robo.li/robo.phar when the Robo team fixes it.
-RUN wget https://github.com/consolidation/Robo/releases/download/1.2.1/robo.phar
+RUN wget https://robo.li/robo.phar
```
Comment (deviantintegral) [inline Dockerfile:28]: Yay, so glad to see this is fixed!

**Comment by deviantintegral** [comment]
I'm not sure why Circle didn't run tests before - I have forked builds turned on, but it threw an error that "no one is following the project". I did a rebuild, and it looks like it didn't pick up the circle config at all. I'm opening a support request.

**Comment by deviantintegral** [comment]
Opened a bug report at https://discuss.circleci.com/t/forked-pr-builds-not-detecting-configuration/21050

**Comment by dakkusingh** [comment]
@deviantintegral can we merge this in?

**Comment by deviantintegral** [comment]
Indeed, tests passed. Thanks!

**Pull request merged**